mne/baseline.py

```diff
@@ -68,11 +68,22 @@ def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
     if bmin is None:
         imin = 0
     else:
-        imin = int(np.where(times >= bmin)[0][0])
+        imin = np.where(times >= bmin)[0]
+        if len(imin) == 0:
+            raise ValueError('bmin is too large (%s), it exceeds the smallest '
+                             'time value' % (bmin,))
```

Comment:
And maybe 'a' and 'b' in the docstring could be changed to bmin and bmax to make the error message more sensible.